File: php/problem_6.php
<?php
function sum_of_squares($result = 0)
{
  foreach (range(1, 101) as $num) {
    $result += $num ** 2;
  }
  return $result;
}

function square_of_sum($result = 0)
{
  foreach (range(1, 101) as $num) {
    $result += $num;
  }
  return $result ** 2;
}

$sum_square_difference = square_of_sum() - sum_of_squares();

$answer = 25164150;
echo $sum_square_difference." === ".$answer;

File: python/problem_6.py
sum_of_squares = sum(x ** 2 for x in range(1, 101))
square_of_sum = sum(x for x in range(1, 101)) ** 2
sum_square_difference = square_of_sum - sum_of_squares

print(True if sum_square_difference == 25164150 else False)
